app/src/main/java/org/breezyweather/settings/preference/composables/SwitchPreference.kt

```diff
@@ -84,6 +89,7 @@ fun SwitchPreferenceView(
     withState: Boolean = true,
     enabled: Boolean = true,
     card: Boolean = true,
+    colors: ListItemColors = ListItemDefaults.colors(AlertDialogDefaults.containerColor),
```

Comment:
The default here should also be `ListItemDefaults.colors()` in my opinion.

Reply:
Agreed. Changes in latest commit.